examples/configs/pose_graph/pose_graph_synthetic.yaml

```diff
@@ -23,6 +23,7 @@ inner_optim:
   ratio_known_poses: 0.1
   reg_w: 1e-3
   vectorize: true
+  empty_cuda_cache: false
```

Comment:
I would like to add `empty_cuda_cache=cfg.solver_device=='cuda' and cfg.inner_optim.solver=='dense'` to `pose_graph_synthetic.py`